internal/gps/vcs_source_test.go

```diff
@@ -15,14 +16,15 @@ import (
 	"testing"
 )
 
+var Strict *bool = flag.Bool("strict", false, "Fail tests when a required binary is missing, instead of skipping")
```

Comment:
actually, also, let's change the name to something specific to the task at hand, rather than the general concept of "strict" (even at the cost of a longer flag name)